HackerRank/Python/Data Types/Find second largest number in a list.py

```diff
@@ -1,25 +1,14 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-'''
-Problem Statement
-
+tatement
 Now, lets delve into one of the most basic data types in python, LIST. You are given 'n' numbers. Store them in a list and find the second largest number. Easy enough!
-
 NOTE : Don't use insertion sort
-
 Input Format
-
 First line contains N. Second line contains Array A[] of N integers each separated by a space.
-
 Output Format
-
 Value of the second largest number.
 '''
 
 N=int(raw_input())
 list=list(set(map(int,raw_input().strip().split(" "))))
 list.sort(reverse=True)
 print list[1]
-
-    
-    
+    print(arr[0])
```

Comment:
remove

Reply:
@ad1992 i have added your code in "Find second largest number in a list.py" and  "Find second largest number in a list-2.py"  file is mine. I was removed .gecici.swp.